resources/views/emails/mcamReport.blade.php

```diff
@@ -0,0 +1,16 @@
+@component('vendor.mail.html.message')
+    The following individuals have qualified for the indicated MCAM for {{$mcamReport['report_date']}}
+
+    @component('mail::table')
+        | Name | Member ID | MCAM Ordinal |
```

Comment:
We can do better than `MCAM Ordinal` 😎 Change to "Award Number"

Reply:
Done!